### ISSUE TITLE
Duda con test 15 Reversal

### ISSUE THREAD

**Issue by piacabrera** [RodrigoToroIcarte/IIC2113-2023-1#199]
Hola! tengo una duda con respecto a un test. Pasa que el error que tengo se da cuando:

Undertaker juega una carta maneuver, y es revertida por Mankind. Dado que el damage de la carta que usa Mankind para el reversal es #, esta debe hacer el daño de la maniobra revertida que en este caso es el daño de BodySlam, que sería 6, pero según el test debería ser 5.

Esto se debe a que la carta originalmente la jugaba Undertaker, y a Mankind le haría 1 menos de daño? es decir, las cartas con # no hacen necesariamente el daño de la carta, sino el daño que le hubiese hecho al oponente?

Muchas gracias :)

**Comment by RodrigoToroIcarte** [comment]
Exacto. Es por culpa de la habilidad de Mankind. El `#` es igual al daño neto que hace la maniobra después de aplicado todos los bonus de daño (ej, habilidad de mankind, Jockeying for Position, etc).

Acá hay más issues relacionados con esta pregunta: https://github.com/RodrigoToroIcarte/IIC2113-2023-1/issues/102 y https://github.com/RodrigoToroIcarte/IIC2113-2023-1/issues/65
